src/acp/util/setVcAccess.ts

```diff
@@ -63,56 +96,63 @@ export const DEFAULT_VC_MATCHER_NAME = "defaultVcMatcher";
  * @param access The access modes to set. Setting a mode to `true` will enable it, to `false`
  * will disable it, and to `undefined` will leave it unchanged compared to what was previously
  * set.
+ * @param options An option object to customize the function behavior:
+ *  - inherit: if set to `true`, the access set to the target resource cascades
+ *    to its contained resources.
  * @returns A copy of the resource and its attached ACR, updated to the new access modes.
  * @since 1.17.0
  */
 export function setVcAccess(
   resourceWithAcr: WithAccessibleAcr,
-  access: Partial<AccessModes>
+  access: Partial<AccessModes>,
+  options: {
+    inherit: boolean;
+  } = { inherit: false }
 ): WithAccessibleAcr {
   let acr = internal_getAcr(resourceWithAcr);
   const defaultVcPolicyIri = `${getSourceIri(acr)}#${DEFAULT_VC_POLICY_NAME}`;
   const defaultVcMatcherIri = `${getSourceIri(acr)}#${DEFAULT_VC_MATCHER_NAME}`;
 
+  const { policy, matcher } = createVcPolicy(
+    acr,
+    defaultVcPolicyIri,
+    defaultVcMatcherIri,
+    access
+  );
+
   let accessControl = getDefaultAccessControlThing(
     resourceWithAcr,
     "defaultAccessControl"
   );
+  if (!getIriAll(accessControl, ACP.apply).includes(asIri(policy))) {
+    // Case when the ACR Thing existed, but did not include a link to the default Access Control.
+    accessControl = addIri(accessControl, ACP.apply, policy);
+  }
```

Comment:
Maybe you want the same condition on the member access setter?

Reply:
My assumption was that potentially duplicating triples wasn't a big deal, because that would result in equivalent RDF graphs, and would get normalized when converted to RDF. Actually I think I'll even remove the condition on the non-member access control, it makes the code a bit easier to read.

Reply:
But in addition I'll add a devX ticket to have some way to specify you want a triple added if it isn't already present.

Reply:
Actually my initial proposal is probably a bad idea, let's show intent first and do as you suggested